CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
 * Add support for `spec.rack.topologyKey` property in Mirror Maker 2 to enable "fetch from the closest replica" feature.
 * Support for the s390x platform
   _(The s390x support is currently considered as experimental. We are not aware of any issues, but the s390x build doesn't at this point undergo the same level of testing as the AMD64 container images.)_
+* Use CustomResource existing spec and status to fix Quarkus native build's serialization
+* Upgrade Builder annotations to 0.90.4
```

Comment:
I don't think this is needed in the CHANGELOG. The first bullet point is fine and makes sense. But I think this one is not needed.